### ISSUE TITLE
Donating Funds to Ratatui

### ISSUE THREAD

**Issue by Lfada** [ratatui/ratatui#611]
Hi Ratatui, I'm Lucas, a core contributor to the Radworks project. https://radworks.org/

[[Radicle](http://www.radicle.xyz/)] has been using https://github.com/ratatui-org/ratatui. It's in fact one of our most critical dependencies!

All Radicle’s core-contributors really appreciate the work you all do and we want to make sure that you are all well-funded to keep doing it. For that reason, we have decided to donate some funds to your project through [[Drips](https://www.drips.network/)](an initiative that is coming out of our organization for funding FOSS).

Who can I talk to about it?

cc: @fdehau @joshka @orhun 

## Problem
None, you've done amazing work. This is a thank you. 

## Solution
Claim funds. Engage on a call to understand what we are trying to do here. If you don't need 100% of the funds, share with your own critical dependencies.

**Comment by orhun** [comment]
Hello Lucas, nice to meet you!

We are currently re-considering to open up funding for `ratatui`: https://github.com/ratatui-org/ratatui/discussions/206#discussioncomment-7470164

Although we're possibly going to go with a different solution, I am happy to discuss the possibility of accepting funding through `drips`, feel free to [join our Discord server](https://discord.gg/pMCEU9hNEj) and ping me there (`@orhunp`) so that we can get the conversation going!

Thank you!

**Comment by joshka** [comment]
Closing out as complete - thanks @Lfada.

**Comment by orhun** [comment]
https://blog.orhun.dev/open-source-funding-with-ratatui/

**Comment by oliverpool** [comment]
Apart from the large transfer last year, I don’t see any other transactions from Drips: https://opencollective.com/ratatui/transactions?searchTerm=Drips&kind=ADDED_FUNDS&kind=CONTRIBUTION&kind=EXPENSE&kind=BALANCE_TRANSFER

Did you receive funds by some other mean?

**Comment by orhun** [comment]
Yeah, we only transferred once from Drips and didn't do any follow ups after that. We probably need to re-claim the project there and re-do the whole process again. (since the organization is renamed to ratatui)

**Comment by joshka** [comment]
>We probably need to re-claim the project there and re-do the whole process again

I don't think there were more funds. It was 10% of the 267k radworks donated total across their deps (though this was worth about 36k when this was converted to open collective because crypto).
